CHANGELOG.md

```diff
@@ -4,9 +4,32 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+
+## [0.1.4][] - 2022-08-15
+
+[0.1.4]: https://github.com/chaostoolkit-incubator/chaostoolkit-wiremock/compare/0.1.3...0.1.4
+
+### Added
+
+- add new actions `update_mappings_status_code_and_body`, `update_mappings_fault`, `delete_all_mappings`, `reset_mappings`
+- update mappings filtering function to filter by any metadata in stub mappings (non-strict filtering). Strict (legacy) filtering will still be used by default for backward compatibility
+- improved stubs mapping filter available for existing `delete_mappings`, `fixed_delay` actions
+- now wiremock full `url` can be used in configuration instead of `host` and `port` combination. Useful when wiremock URL uses https
+- updated wiremock version used in continuous integration to latest stable `2.33.2`
+- updated dependency management using **Pipenv**. `requirements.txt` and `requirements-dev.txt` files can now be generated from requirements lock file with `make requirements` (Python >= 3.9 only)
+- general code formatting using **black** and linting fixes
+- add Dockerfile to run linting and unit tests locally
```

Comment:
curious why not rely on the local python (venv or not) rather than going through a container?

Reply:
We created a container because the build pipeline in Travis.ci uses python 3.7 and we cannot install it in our company laptops, hence the only way to test this is using a container